### PR TITLE
Fix ProcessDetailQuery by removing form subfields

### PR DESCRIPTION
**Pull request by tjeerddie**
- `form` changed from graphql type to a JSON in the backend.

### CODE REVIEW

**Comment by DutchBen** [comment]
Can you add the ticket in the branch name and the pr description.

File: packages/orchestrator-ui-components/src/graphqlQueries/processDetailQuery.ts [hunk 1]
```diff
@@ -20,14 +20,7 @@ export const GET_PROCESS_DETAIL_GRAPHQL_QUERY: TypedDocumentNode<
...
                 isTask
-                form {
-                    additionalProperties
-                    definitions
-                    properties
-                    required
-                    title
-                    type
-                }
+                form
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/graphqlQueries/processDetailQuery.ts:23]: Does this mean all these props are still there you just dont need to request explictly request them?

**Review by DutchBen — APPROVED** [review]

**Review by wouter1975 — APPROVED** [review]

File: packages/orchestrator-ui-components/src/graphqlQueries/processDetailQuery.ts (hunk 1 above)
Comment (tjeerddie) [reply packages/orchestrator-ui-components/src/graphqlQueries/processDetailQuery.ts:23]: yes and you can't request them explicitly anymore

Comment (tjeerddie) [reply packages/orchestrator-ui-components/src/graphqlQueries/processDetailQuery.ts:23]: going to create a backend 2.1.0 release and add a changeset that this requires backend 2.1.0

**Comment by heanet** [comment]
Hi, this seems to have introduced an error for us, when trying to access a workflow details page:

```
2024-02-12 10:18:21 [error    ] Field 'form' of type 'ProcessFormType' must have a selection of subfields. Did you mean 'form { ... }'?

GraphQL request:12:7
11 |       isTask
12 |       form
   |       ^
13 |       steps { [strawberry.execution] 
INFO:     10.0.7.33:50057 - "POST /api/graphql HTTP/1.1" 200 OK
```

Is there a further change we need to make to prevent this?

**Comment by tjeerddie** [comment]
> Hi, this seems to have introduced an error for us, when trying to access a workflow details page:
> 
> ```
> 2024-02-12 10:18:21 [error    ] Field 'form' of type 'ProcessFormType' must have a selection of subfields. Did you mean 'form { ... }'?
> 
> GraphQL request:12:7
> 11 |       isTask
> 12 |       form
>    |       ^
> 13 |       steps { [strawberry.execution] 
> INFO:     10.0.7.33:50057 - "POST /api/graphql HTTP/1.1" 200 OK
> ```
> 
> Is there a further change we need to make to prevent this?

orchestrator core 2.1.0 had changed form graphql type to JSON and broke frontend, this fixed that but breaks with previous backend versions. https://github.com/workfloworchestrator/orchestrator-ui-library/releases/tag/%40orchestrator-ui%2Forchestrator-ui-components%400.6.0

**Comment by heanet** [comment]
Thanks - paying the price for living on the haemorrhaging edge! We'll look at pinning our version instead of pulling latest so that we can read release notes as we go.

**Pull request merged**